### PR TITLE
[textanalytics] regen on v3.1

### PR DESCRIPTION
**Pull request by kristapratico**
Regenerates and records with v3.1

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_1/aio/_text_analytics_client.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ async def _send_request(self, http_request: HttpRequest, **kwargs: Any) -> Async
         :rtype: ~azure.core.pipeline.transport.AsyncHttpResponse
         """
         path_format_arguments = {
+            'ApiVersion': self._serialize.url("api_version", api_version, 'str', skip_quote=True),
```
Comment (kristapratico) [inline sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_1/aio/_text_analytics_client.py:60]: Not sure if this is handling the new ApiVersion parameterizing in the swagger correctly - `api_version` does not exist in this scope.

Comment (kristapratico) [reply sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_1/aio/_text_analytics_client.py]: swagger transform fixes this for now

**Review by iscai-msft — APPROVED** [review]
bellisimo!

**Pull request merged**